refactor/client_side/interface_with_covalent.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+
+"""Functions to dispatch a workflow and get result of a workflow"""
+
+# Include a download to file parameter to allow user to save the result in a file if required
+
+from copy import deepcopy
+from functools import wraps
+from io import BytesIO
+
+import cloudpickle as pickle
+import requests
+
+from covalent._results_manager.result import Result
+from covalent._workflow.lattice import Lattice
+
+SUBMIT_DISPATCH_QUEUER_ADDR = "http://localhost:8000/api/v0/submit/dispatch"
```

Comment:
Use the Settings class and `get_route` method. No more hardcoded file paths.

Reply:
Updated.